### PR TITLE
https://github.com/intel-analytics/analytics-zoo/issues/751

### PR DESCRIPTION
**Pull request by xiaxin1998**

### CODE REVIEW

**Comment by hkvision** [comment]
Revert the changes for the notebook

File: apps/anomaly-detection/README.md [hunk 1]
```diff
@@ -5,8 +5,16 @@ This is a simple example of unsupervised anomaly detection using Analytics Zoo K
...
+
+## Run Jupyter after pip install
+```bash
+export SPARK_DRIVER_MEMORY=2g
+jupyter notebook --notebook-dir=./ --ip=* --no-browser
+```
+
+## Run Jupyter with prebuilt package
 * Run `export SPARK_HOME=the root directory of Spark`.
 * Run `export ANALYTICS_ZOO_HOME=the dist directory under the Analytics Zoo project`.
```
Comment (hkvision) [inline apps/anomaly-detection/README.md]: Change this.

**Comment by xiaxin1998** [comment]
fixed issue#751

**Comment by xiaxin1998** [comment]
fixed #751

File: apps/anomaly-detection/README.md [hunk 2]
```diff
@@ -5,10 +5,18 @@ This is a simple example of unsupervised anomaly detection using Analytics Zoo K
...
+## Run Jupyter after pip install
+```bash
+export SPARK_DRIVER_MEMORY=2g
+jupyter notebook --notebook-dir=./ --ip=* --no-browser
+```
+
+## Run Jupyter with prebuilt package
 * Run `export SPARK_HOME=the root directory of Spark`.
-* Run `export ANALYTICS_ZOO_HOME=the dist directory under the Analytics Zoo project`.
+* Run `export ANALYTICS_ZOO_HOME=the folder where you extract the downloaded Analytics Zoo zip package
```
Comment (hkvision) [inline apps/anomaly-detection/README.md]: missing ` at the end

**Comment by hkvision** [comment]
Please revert the changes for ipynb

**Pull request closed without merging**